qiskit_nature/second_q/problems/electronic_properties_container.py

```diff
@@ -0,0 +1,72 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The electronic properties container."""
+
+from __future__ import annotations
+
+from typing import cast
+
+from qiskit_nature.second_q.properties import (
+    AngularMomentum,
+    ElectronicDipoleMoment,
+    Magnetization,
+    ParticleNumber,
+)
+
+from .properties_container import PropertiesContainer
+
+
+class ElectronicPropertiesContainer(PropertiesContainer):
+    """The container class for electronic structure properties."""
+
+    @property
+    def angular_momentum(self) -> AngularMomentum | None:
+        """Returns the :class:`qiskit_nature.second_q.properties.AngularMomentum` property."""
```

Comment:
Minor comment. I am thinking that """Returns the angular momentum property.""" as plain text may suffice and be simpler/easier to read/parse - plus potentially can be translated. The above will put the fully qualified class there as a link and since the return type will be listed as AngularMomentum and will be a link anyway taking one to the exact class maybe too many links - actually at present none of them are working links since properties are not coming out in the docs at present - I noticed same with drivers yesterday where there was a non working link).

Also when the properties are built it uses that in the list of all the fields in the class, as below, which looks a little too much. I think plain text descriptions would be better as the link on the left will bring up the details of exact return type. Maybe I could go with using a ~ so its just the name not the fully qualified one, but in my mind plain textual description there would be preferable I think.

![image](https://user-images.githubusercontent.com/40241007/184220419-98e9ea1c-abbc-4c8a-98da-a01d35a94e57.png)

Same comment applies for all getter/setters like this.

Reply:
Not sure why I missed to add the `~` here, but yes, plain-text docstrings work for me, too.
Will update this tomorrow :+1: